packages/web-component/src/components/read-along-component/read-along.tsx

```diff
@@ -96,11 +96,13 @@ export class ReadAlongComponent {
   @Prop() cssUrl?: string;
 
   /**
-   * Toggle the use of assets folder for resolving urls. Defaults to on
-   * to maintain backwards compatibility
+   * Toggle the use of an assets folder. All image paths
+   * will be prefixed with this. Defaults to 'assets'.
```

Comment:
Not quite accurate, it actually defaults to `'assets/'`, with the slash.

In fact, since you do `this.imageAssetsFolder + path;` in `urlTransform()`, the slash is mandatory. That should be documented if it stays that way, but in fact I would much rather add it automatically if it's not there rather than require it.